types/src/on_chain_config/execution_config.rs

```diff
@@ -70,7 +70,11 @@ impl OnChainExecutionConfig {
     /// Features that are ready for deployment can be enabled here.
     pub fn default_for_genesis() -> Self {
         OnChainExecutionConfig::V4(ExecutionConfigV4 {
-            transaction_shuffler_type: TransactionShufflerType::SenderAwareV2(32),
+            transaction_shuffler_type: TransactionShufflerType::Fairness {
+                sender_conflict_window_size: 256,
```

Comment:
why do we want 32 => 256 increase from before for this one?

Reply:
see @sitalkedia's comment. Updating this will result in the new number being tested in the next devnet release.

Reply:
it looks like realistic_env_max_load  gets more block limit hits with 256..

Reply:
Well, I'll decouple the 32->256 change from this for now and see.